src/GitHub.VisualStudio/Helpers/ActiveDocumentSnapshot.cs

```diff
@@ -19,8 +19,14 @@ public class ActiveDocumentSnapshot : IActiveDocumentSnapshot
         public ActiveDocumentSnapshot([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
         {
             StartLine = EndLine = -1;
-            Name = Services.Dte2?.ActiveDocument?.ProjectItem.Name;
+            Name = Services.Dte2?.ActiveDocument?.FullName;
+            var projectItem = Services.Dte2?.ActiveDocument?.ProjectItem;
 
+            if ((String.Compare(Name, projectItem.FileNames[1], StringComparison.Ordinal) != 0) && (String.Compare(Name, projectItem.FileNames[1], StringComparison.OrdinalIgnoreCase) == 0))
```

Comment:
Right, you could technically only grab the name from `FileNames` if the casing is wrong in `ActiveDocument.FullName`, but in practice, `FileNames` always has the correct case while `ActiveDocument.FullName` sometimes has the wrong case, which means we might as well trust `FileNames` always and only do one comparison to confirm that the `FileNames` entry is the correct one.